Automattic/jetpack
# Revert "Premium Content / Subscriptions: introduce Paid Newsletter functionality

#### Changes proposed in this Pull Request:

* This reverts the changes made in this PR: https://github.com/Automattic/jetpack/pull/26417
* This was done as the WoA deploy resulted in a large number of fatals errors, specifically:

```
Uncaught ArgumentCountError: Too few arguments to function Automattic\Jetpack\Extensions\Subscriptions\jetpack_filter_excerpt_for_newsletter(), 1 passed
```
and

```
Uncaught ArgumentCountError: Too few arguments to function Automattic\Jetpack\Extensions\Premium_Content\Subscription_Service\Token_Subscription_Service::visitor_can_view_content(), 1 passed
```

... with two expected in each case.


#### Other information:


#### Jetpack product discussion

p9o2xV-2HN-p2#comment-6910
p1672250536716759/1672247705.462969-slack-C01U2KGS2PQ


#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* Apply this PR and make sure the previous (prior to the linked PR) subscriptions / premium content functionality is not impacted in any way.
